extensions/oidc-client-filter/runtime/src/main/java/io/quarkus/oidc/client/filter/runtime/OidcClientFilterConfig.java

```diff
@@ -11,12 +11,16 @@ public class OidcClientFilterConfig {
     /**
      * Enable OidcClientRequestFilter for all the injected MP RestClient implementations.
      * If this property is disabled then OidcClientRequestFilter has to be registered as an MP RestClient provider.
+     * You can use this property in combination with the `@OidcClientFilter` annotation, as sometimes, you might need to
+     * specify named OidcClient like this `@OidcClientFilter("jwt-secret")` for individual Rest client,
+     * while the rest of the clients will default to the configured client.
      */
     @ConfigItem(defaultValue = "false")
     public boolean registerFilter;
 
     /**
-     * Name of the configured OidcClient.
+     * Name of the configured OidcClient used by the OidcClientRequestFilter. You can override this configuration for
+     * individual Rest clients with the `@OidcClientFilter` annotation like this: `@OidcClientFilter("jwt-secret")`.
```

Comment:
```suggestion
     * individual MO RestClient with the `io.quarkus.oidc.client.filter.OidcClientFilter` annotation.
```

Reply:
`MP RestClient` is used in other parts of these docs so just propose to use the same term. And IMHO if we do want to show the example here then we should do it with imports, client like jwt-secret config, so it is probably easier to handle in the tutorial/ref guide